x-pack/plugins/ml/public/application/components/anomalies_table/anomalies_table_columns.js

```diff
@@ -116,19 +116,22 @@ export function getColumns(
       field: 'severity',
       'data-test-subj': 'mlAnomaliesListColumnSeverity',
       name: (
-        <EuiToolTip
-          content={i18n.translate('xpack.ml.overview.anomalyDetection.tableSeverityTooltip', {
-            defaultMessage:
-              'A normalized score between 0-100, which indicates the relative significance of the anomaly record results.',
+        <span>
```

Comment:
@alexwizp These changes LGTM but curious if we should create an issue for `EuiToolTip` if accessibility/ability to focus is an issue?

Reply:
I'm not sure that we have an issue on the EUI side. I think they are trying to keep a balance between being generic and having a clear API. The `EuiIconTooltip` was created to address these kinds of cases. 

But maybe @cee-chen can give more context here.

Reply:
Alexey said it perfectly! ❤️